src/Dev/Deprecation.php

```diff
@@ -137,30 +145,38 @@ protected static function get_called_method_from_trace($backtrace, $level = 1)
     }
 
     /**
-     * Determine if deprecation notices should be displayed
+     * This method is no longer used
      *
      * @return bool
+     * @deprecated 4.12.0 Will be removed without equivalent functionality to replace it
      */
     public static function get_enabled()
     {
-        // Deprecation is only available on dev
+        static::notice('4.12.0', 'Will be removed without equivalent functionality to replace it');
+        // noop
+    }
+
+    private static function get_is_enabled()
+    {
         if (!Director::isDev()) {
             return false;
         }
-        if (isset(self::$enabled)) {
-            return self::$enabled;
+        if (Environment::getEnv('SS_DEPRECATION_ENABLED')) {
+            return true;
         }
-        return Environment::getEnv('SS_DEPRECATION_ENABLED') ?: true;
+        return static::$is_enabled;
     }
 
     /**
-     * Toggle on or off deprecation notices. Will be ignored in live.
+     * This method is no longer used
      *
      * @param bool $enabled
+     * @deprecated 4.12.0 Use enable() instead
```

Comment:
```suggestion
     * @deprecated 4.12.0 Use disable() instead
```
Same change needed for the notice below

Reply:
It should be enable()? That class defaults to 'off', so you'd realistically only every call enable(), not disable(). The only reason I included disable() was to switch this off in DeprecationTest::tearDown() in case of any unexpected side effects of leaving it on

Reply:
Yup, for some reason I thought the old method was called "set_disabled".... my bad